apps/verticalhome/js/pinned_apps.js

```diff
@@ -129,21 +129,46 @@
 
     init: function () {
       this.onLoadSettings();
+      this.items[0].element.classList.add('middle');
     },
 
     onLoadSettings: function() {
       var pinnedAppsList = app.getPinnedAppsList();
+      if(!document.getElementById('moreApps')){
+        var moreApps = document.createElement('div');
+        moreApps.className = 'pinned-app-item';
+        moreApps.setAttribute('data-index', '999');
+        moreApps.setAttribute('id', 'moreApps');
+        moreApps.innerHTML = '<input id="moreAppsButton" type="button"/>'+
+          '<span class="title">More Apps</span>';
```

Comment:
"writing to innerHTML will cause trouble on a security audit. It is okay to set innerHTML to the empty string to delete elements, but setting this to a string, especially something that is not a string literal is considered a risk of bugs." :copyright: **(David)**

Do this through document.createElement. :sake: